### PR TITLE
Crypto utils tests - ref #287

### PR DESCRIPTION
**Pull request by estefysc**

### CODE REVIEW

**Comment by muke1908** [comment]
Hi @estefysc , take a look. Now it's passing. You can add the other UT now.

**Comment by estefysc** [comment]
Hey, @muke1908. Checking now. Thanks for the help.

**Comment by estefysc** [comment]
@muke1908 ready for review. Not too sure these are the best. Lest me know,

File: service/src/crypto.test.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import e from "cors";
```
Comment (muke1908) [inline service/src/crypto.test.ts:1]: redundant
```suggestion

```

**Review by muke1908 — APPROVED** [review]

**Comment by muke1908** [comment]
> @muke1908 ready for review. Not too sure these are the best. Lest me know,

Thanks for the contribution! These UTs are enough to test the basic crypto module.

**Pull request merged**